docs/contribute.rst

```diff
@@ -324,25 +348,55 @@ whether given functionality is already available in their package:
 
     .. versionadded:: 1.23
 
-If the pull request addresses an existing issue, mention it using
-one of the automatically parsed formats so that it is linked to
-it, for example:
+.. _review:
 
-.. code-block:: markdown
+Review
+------------------------------------------------------------------
 
-    Fix #1234.
+Code review is an essential part of the workflow. It ensures good
+quality of the code and prevents introducing regressions, but it
+also brings some additional benefits: By reading code written by
+others you can learn new stuff and get inspired for your own code.
+Each completed pull request review helps you, little by little, to
+get familar with larger part of the project code and empowers you
+to contribute more easily in the future.
 
-By default only a core set of tests is executed against a newly
-created pull request and its updates to verify basic sanity of the
-change. Once the pull request content is ready for a thorough
-testing add the ``full test`` label and make sure that the
-``discuss`` label is not present. All future changes of the pull
-request will be tested with the full test coverage. For changes
-related to documentation only the full test suite is not required.
+For instructions how to locally try a change on your laptop see
+the :ref:`develop` section. Basically just enable the development
+environment and check out the pull request branch or use the
+`github cli`__ to check out code from a fork repository:
```

Comment:
I personally sometimes install the rpm built by packit inside PRs to review the changes. Would it make sense to mention this possibility here as well?

Reply:
Good point, added in a7d1504c.